test/fixtures/scp/bin/ssh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+for arg in "$@"; do
+    case "$arg" in
+	-o)
+		shift 2;;
+	local)
+		shift 1;;
```

Comment:
Did this, the reason this passed CI is because shfmt doesn't run on fixtures, added a commit to make both shfmt and shellcheck run on fixtures/*/bin/*

Reply:
Thanks! I'm unsure if all `fixtures/*/bin/*` would be written in shell language in the future, but it should be fine. We can adjust it when we add a new executable file not written in shell language.